pyanaconda/ui/gui/spokes/network.py

```diff
@@ -1521,11 +1521,14 @@ def apply(self):
                 log.debug("network spoke (apply) refresh payload")
 
                 from pyanaconda.payload import payloadMgr
-                if not self.payload.verifyAvailableRepositories():
+                if payloadMgr.running:
+                    log.debug("Payload is in the process of restarting, skip the repo availability check.")
```

Comment:
What if network settings changed since the payload restart has been triggered ? I guess the chance is small, but I guess we might get false results if the network using part of the check did run before network configuration has been changed. That could be solved by scheduling an extra check, but I'm not sure if that's not an overkill.